docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Monitor recent activity logged actions.
+---
+
+# Recent activity log
+
+## Activity list
+
+On the Back Office, in **Admin** -> **Activity list**, you can consult what actions were taken by users recently.
+
+!!! note
+
+    - To see the Admin menu, you must have the "Setup / Administrate" [permission](permission_system.md).
+    - To see the Activity list, you must have the "Activity Log / Read" permission (this permission might be limitated to only see your own activity).
+
+By default, the following actions are logged:
+
+- [Content](content_items.md) [create](create_edit_content_items.md#create-content-items), [publish](publish_instantly.md#publish-content-instantly), update, trash, recover, [delete](copy_move_hide_content.md#multi-file-move), delete translation, hide and reveal
+- Location create, delete, [hide](manage_locations_urls.md#hide-locations), reveal, update, move, [swap](manage_locations_urls.md#swap-locations) and subtree copy
+- Product [create](create_edit_product.md), update and delete
```

Comment:
```suggestion
- Product [create](create_edit_product.md), update, and delete
```

Reply:
All missing commas added in dc582cd6591b632199c01a4b94e3440477566498